openeogeotrellis/integrations/s3proxy/exceptions.py

```diff
@@ -0,0 +1,14 @@
+class ProxyException(RuntimeError):
+    ...
```

Comment:
FYI: for exceptions it's more common to use `pass` (which explicitly means: do nothing)  instead of `...`  (which gives off work-in-progress vibes or even "must be implemented by subclasses" )

Reply:
Will change remaining `...`